tun/src/unix/linux/mod.rs

```diff
@@ -4,10 +4,12 @@ use socket2::{Domain, SockAddr, Socket, Type};
 use std::fs::OpenOptions;
 use std::io::{Error, Write};
 use std::mem;
-use std::net::{Ipv4Addr, Ipv6Addr, SocketAddrV4, SocketAddrV6};
```

Comment:
Looks like VSCode is doing this automatically for me (I'm using Codespaces). I think it's doing that because we aren't using SocketAddrV6 at the moment.